New events documentation (#760)
Altinn/altinn-studio-docs: content/events/publish-events/_index.en.md

---
title: Publish events
linktitle: Publish events
description: Documentation for event publishers
weight: 20
---

Events published to Altinn are pushed to all authorized subscribers and persisted for on-demand retrieval for 90 days. 

> Cloud event spec is followed, but additional Altinn extension properties are defined that you should know 

Reviewer: ```suggestion
The CloudEvent specification is followed, but there are additional Altinn specific properties that you should know about
```

and use where you see fit.

> - resource (required)

Reviewer: ```suggestion
- resource (always required)
```
